docs/hub/models-inference.md

```diff
@@ -1,30 +1,76 @@
 # Inference Providers
 
-Please refer to the [Inference Providers Documentation](https://huggingface.co/docs/inference-providers) for detailed information.
+Hugging Face's model pages have free inference for thousands of models, so you can try them all out right in the browser. It's also powered by Inference Providers.
```

Comment:
not free, it's paid

Reply:
```suggestion
Hugging Face's model pages have pay-as-you-go inference for thousands of models, so you can try them all out right in the browser. Service is powered by Inference Providers and includes a free-tier.
```

(maybe like this to mention it's paid with a free-tier?)